recipes/openjph/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+import os
+
+required_conan_version = ">=1.53.0"
+
+class OpenJPH(ConanFile):
+    name = "openjph"
+    description = "Open-source implementation of JPEG2000 Part-15 (or JPH or HTJ2K)"
+    license = "BSD-2-Clause"
+    homepage = "https://github.com/aous72/OpenJPH"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("ht-j2k", "jpeg2000", "jp2", "openjph", "image", "multimedia", "format", "graphics")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_tiff": [True, False],
```

Comment:
What happens when a CI that supports some SIMD options builds this with the default options here for all extensions https://github.com/aous72/OpenJPH/blob/master/CMakeLists.txt#L29, but a consumer without that support tries to run the same package? The package id does not change